factcast-site/documentation/content/factus/snapshot-projections.md

```diff
@@ -42,6 +42,10 @@ public class UserNames implements SnapshotProjection {
 
 This projection is interested in `UserCreated` and `UserDeleted` EventObjects and can be serialized by the `SnapshotSerializer`.
 
+**Attention** - when the projection class is changed (e.g. a field is added), there will be a problem with deserialisation.
```

Comment:
adding a field is not a problem. maybe we should use renaming here as a breaking example